### DISCUSSION TITLE
Welcome to Rubberduck Discussions!

### DISCUSSION THREAD

**Discussion by retailcoder** [rubberduck-vba/Rubberduck#5923] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SmileyFtW** [comment]
My name is David Miley and found out about RD in the Excel-L list group. I am retired but still do Excel VBA for fun as well as for a golf course supporting their sales analysis. I am a licensed HAM radio operator in North Central Texas; there is always something HAM related that I find myself doing VBA in Excel for.

**Comment by NguoiHaiPhong** [comment]
Hello. I have an idea a bout improve Rubberduck
Can you create custom VBA Editor like  addin "Code manager.net 2016 for MS Excel"
![image](https://user-images.githubusercontent.com/106435381/229045636-31780119-4bc2-4e51-88be-1bfa10145a4e.png)

**Reply by retailcoder** [reply]
Hi! You're pretty much describing Rubberduck 3.0 here - that's indeed the direction we're headed towards. You can read more about how RD3 is coming along on the project's blog, website, and Discord server. Cheers!
